src/shared/components/mutationTable/column/ProteinChangeColumnFormatter.tsx

```diff
@@ -125,4 +127,28 @@ export default class ProteinChangeColumnFormatter
                 <span>{text}</span>
         );
     }
+
+    public static renderWithMutationStatus(data:Mutation[])
```

Comment:
Merged two different `ProteinChangeColumFormatter` files into this one, since now both tables use the same renderer.